components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -3791,6 +3791,103 @@ func TestListRulesForProject(t *testing.T) {
 			assert.NoError(t, err)
 			assert.Zero(t, len(resp))
 		}},
+		{"when there are staged changes for the project's rules, returns the latest changes to the rules", func(t *testing.T) {
```

Comment:
I think the phrasing around "latest" is maybe too ambiguous. What about 
"when there are only staged changes for the project's rules, returns the staged changes to the rules" 
and
"when there are staged changes for some of the project's rules, returns those staged versions and rules that have no staged changes"